### PR TITLE
Send zoom status for dummy too

### PR DESCRIPTION
**Pull request by def-**

### CODE REVIEW

**Comment by heinrich5991** [comment]
Conflicts from #2471.

**Comment by def-** [comment]
Fixed.

File: src/game/client/gameclient.cpp [hunk 1]
```diff
@@ -1552,21 +1552,28 @@ void CGameClient::OnNewSnapshot()
...
+		CMsgPacker Packer(Msg.MsgID(), false);
+		Msg.Pack(&Packer);
+		if(ZoomToSend != LastZoom)
+			Client()->SendMsgY(&Packer, MSGFLAG_VITAL, 0);
+		if(Client()->DummyConnected())
+			Client()->SendMsgY(&Packer, MSGFLAG_VITAL, 1);
+		LastZoom = ZoomToSend;
+		LastScreenAspect = Graphics()->ScreenAspect();
 	}
+	LastDummyConnected = Client()->DummyConnected();;
```
Comment (fokkonaut) [inline src/game/client/gameclient.cpp:1576]: ;;

Comment (def-) [reply src/game/client/gameclient.cpp:1576]: Done.

**Comment by heinrich5991** [comment]
bors r+

Thanks.

**Pull request merged**